### DISCUSSION TITLE
Add static HTML page | or managing html <HEAD> section

### DISCUSSION THREAD

**Discussion by serelsnauw** [Budibase/budibase#8287] [Ideas]
Hi all,

I'm trying to embed some location-based AR components (mind-AR-js) into Budibase. For this, I must embed some scripts in the < head > section, which the "embed" component in Budibase can't do. 
I've got a working static HTML page which works just fine. All I would like is adding / managing this page in Budibase (and if possible let's me talk to some data sources for multiple database-fed GPS locations :-P ), or adding some < HEAD > scripts to a screen.

Anybody got a clue, or would this just be a nice feature? :-)
Thanks in advance, Serel.

**Comment by melohagan** [comment]
Hey @serelsnauw 

Why not try creating a custom component instead? It'll give you much more control and be easier to maintain and share: https://docs.budibase.com/docs/custom-plugin

**Reply by serelsnauw** [reply]
Hi and thanks @melohagan 
I did not think of that yet (as plugins are quite new to me), so let's start with that :-)

Again, thanks for you're help, much appreciated!

**Reply by melohagan** [reply]
You're welcome, and best of luck! If you have any questions feel free to ask here or in the discord.
